lib/utils/fetchSDP.tsx

```diff
@@ -0,0 +1,29 @@
+import { WsSdpPipeline, VideoMedia } from 'media-stream-library'
+
+export const fetchSDP = (wsURI: string, rtspURI: string) => {
+  const pipeline = new WsSdpPipeline({
+    ws: { uri: wsURI },
+    rtsp: { uri: rtspURI },
+  })
+
+  return pipeline.sdp
+    .then((sdp) => {
+      return sdp
+    })
```

Comment:
Does this do anything?

Reply:
The sdp object contains more data than the transformation matrix. So if a user wants to build upon that, then it's available.

Reply:
Yeah, but just removing it should return the same thing? The `then` looks like just a passthrough?